test/performance/odp_queue_perf.c

```diff
@@ -470,11 +473,22 @@ static int run_test(void *arg)
 		do {
 			src_queue = src_queue_tbl[queue_idx];
 			dst_queue = dst_queue_tbl[queue_idx];
+			num_ev = 0;
 
 			queue_idx++;
 			if (queue_idx == num_queue)
 				queue_idx = 0;
 
+			if (single) {
```

Comment:
I wonder if it is ok to have this check here inside the polling loops. As the condition never changes, it could be moved outside the loop by having separate loops for the single and multi case and then the test loop overhead would be a tiny bit smaller. test_run() could be changed to an always-inlined function that takes 'single' as a parameter and then a new test_run could test for 'single' and call the function with a constant parameter so that the compiler would remove the condition testing from inside the loop.